### DISCUSSION TITLE
More sequence.fasta？

### DISCUSSION THREAD

**Discussion by Xoey0314** [Kinggerm/GetOrganelle#255] [Q&A]
Dear @Kinggerm ,
Thank you for developing such a great software.
I am writing to you to inquire about a question related to the use of the GetOrganelle software for chloroplast genome assembly. I have installed and used the software without any errors. However, regardless of whether I use resequencing data or deep sequencing data, more than two sequence files are outputted as results (although the instructions often mention that only two files will be generated). Each sequence file has the same size. Is this normal? Can I choose any of these files? Thank you very much.

The output files are as follows:
xinxin	152330	March	30	10:14	embplant_pt.K115.complete.graph1.1.repeat_pattern1.path_sequence.fasta
xinxin	152330	March	30	10:14	embplant_pt.K115.complete.graph1.2.repeat_pattern1.path_sequence.fasta
xinxin	152330	March	30	10:14	embplant_pt.K115.complete.graph1.3.repeat_pattern2.path_sequence.fasta
xinxin	152330	March	30	10:14	embplant_pt.K115.complete.graph1.4.repeat_pattern2.path_sequence.fasta
xinxin	152330	March	30	10:14	embplant_pt.K115.complete.graph1.5.repeat_pattern2.path_sequence.fasta
xinxin	152330	March	30	10:14	embplant_pt.K115.complete.graph1.6.repeat_pattern2.path_sequence.fasta
xinxin	127678	March	30	10:14	embplant_pt.K115.complete.graph1.selected_graph.gfa
xinxin	90998118	March	30	10:12	extended_1_paired.fq
xinxin	7617243	March	30	10:12	extended_1_unpaired.fq
xinxin	90998118	March	30	10:12	extended_2_paired.fq
xinxin	3766629	March	30	10:12	extended_2_unpaired.fq
xinxin	397863	March	30	10:14	extended_K115.assembly_graph.fastg
xinxin	6908	March	30	10:14	extended_K115.assembly_graph.fastg.extend-embplant_pt-embplant_mt.csv
xinxin	390624	March	30	10:14	extended_K115.assembly_graph.fastg.extend-embplant_pt-embplant_mt.fastg
xinxin	4096	March	30	10:14	extended_spades
xinxin	9062	March	30	10:14	get_org.log.txt
xinxin	4096	March	30	9:52	seed

Best regards, Xinxin

**Comment by Kinggerm** [comment]
Please see https://github.com/Kinggerm/GetOrganelle/wiki/FAQ#why-are-there-so-many-path_sequencefasta-files
Let me know if it makes sense to you.

Also, this is not an issue. I'm converting it into a discussion.
